cirq/protocols/commutes_protocol.py

```diff
@@ -0,0 +1,223 @@
+# Copyright 2018 The Cirq Developers
```

Comment:
```suggestion
# Copyright 2019 The Cirq Developers
```

Reply:
Done. We generally don't worry about these though.